README.md

```diff
@@ -35,6 +35,39 @@ Each platform maintains multiple types of releases:
 * Beta: A version that is nearing full "stable" release.
 * Nightly: Contains the very latest code. These versions are published daily whenever the code changes.
 
+# Contributing
```

Comment:
This needs to be clear that it's only about distribution of npm packages -- right now it looks like it's about contributing to the Keyman project.

I do wonder if it should be somewhere outside the main project README.md because majority of contributors won't be working with them directly. Perhaps in a new docs/ folder.

Reply:
Yeah, I was wondering about that since none of the other stuff in the main README has  documentation for @keymanapp developers... Where are we supposed to keep that?

Reply:
A lot of the dev documentation is in the [keymanapp/keyman wiki](https://github.com/keymanapp/keyman/wiki). Mostly because it was in constant flux and it was a hassle to have it generating PRs etc. Advantage of the wiki also is it keeps the docs out of the alpha/beta/stable release cycle. We have a similar problem with the version history maintenance. So that's another appropriate place for it, perhaps.